client/components/exercise-info-accordion.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export default class ExerciseInfoAccordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { exerciseInfo: {} };
+  }
+
+  componentDidMount() {
+    const exerciseID = this.props.exerciseid;
+    fetch(`/api/exercise-by-id/${exerciseID}`)
+      .then(result => result.json())
+      .then(exerciseInfo => {
+        this.setState({ exerciseInfo });
+      });
+  }
+
+  render() {
+    let { images, description } = this.state.exerciseInfo;
+    if (!images) {
+      images = [];
+      images.push('', '');
+    }
+    return (
+      <div className="exercise-details">
+        <img src={images[0]}></img>
+        <img src={images[1]}></img>
+        <div className="exercise-steps">
+          <h2>STEPS</h2>
+          <p>{description}</p>
```

Comment:
Since your API is sending you HTML, you need to use a combination of 
- [`DOMPurify`](https://www.npmjs.com/package/dompurify)
- [`dangerouslySetInnerHTML`](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml)

Setting `innerHTML` is dangerous, but `DOMPurify` is meant to keep it safe by stripping out potential XSS attacks.